pulsar-broker/src/test/java/org/apache/pulsar/broker/admin/TopicPoliciesTest.java

```diff
@@ -1958,8 +1968,25 @@ private void doTestTopicMaxMessageSize(boolean isPartitioned) throws Exception {
         assertNull(admin.topicPolicies().getMaxMessageSize(topic));
         // set msg size
         admin.topicPolicies().setMaxMessageSize(topic, 10);
-        Awaitility.await().until(()
-                -> pulsar.getTopicPoliciesService().getTopicPolicies(TopicName.get(topic)) != null);
```

Comment:
We should not remove this judgment. Even if it is a partitioned topic, only the topic without -partition- will be saved in the Topic Policy.

For example: persistent://tenant/namespace/topic-partition has 10 partitions, only one copy of data will be saved in Topic Policy, and the key is persistent://tenant/namespace/topic-partition

Reply:
Added.